.github/workflows/coverage.yaml

```diff
@@ -50,6 +54,9 @@ jobs:
     needs: [coverage]
     steps:
       - uses: actions/github-script@v6
+        env:
+          issue_number: ${{ github.event.issue.number }}
+        if: ${{ env.issue_number }}
```

Comment:
This prevents the github-script from running on pushes that don't have pull requests and thus don't have a place to leave a comment.

It's possible to write this logic in javascript (I did this for testing), but I don't see the point as it's much more expensive to have the poor runner actually use the runtime than it is to get the host to evaluate this eagerly. -- It'd make more sense to do it in the code if you could properly produce a step summary from the script (see below)